itou/api/applicants_api/perms.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework.permissions import IsAuthenticated
+
+
+class ApplicantsAPIPermission(IsAuthenticated):
+    def has_permission(self, request, view) -> bool:
+        """
+        Check that user:
+            - belongs to one SIAE only (dedicated API account)
+            - is admin
+        """
+        if not super().has_permission(request, view):
+            return False
+
+        memberships = request.user.active_or_in_grace_period_siae_memberships()
+        if not memberships or len(memberships) > 1:
+            # Unable to filter on a single SIAE or no SIAE for this user at all
+            return False
+
+        return memberships.first().is_admin
```

Comment:
Un peu plus clair en version [EAFP](https://docs.python.org/3/glossary.html#term-EAFP) à mon avis.
```suggestion
        try:
            return memberships.get().is_admin
        except SiaeMembership.DoesNotExist, SiaeMembership.MultipleObjectsReturned:
            return False
```